### DISCUSSION TITLE
Image files not being displayed on the HTML render.

### DISCUSSION THREAD

**Discussion by igrzi** [marp-team/marp#493] [Q&A] [answered]
Hi, I'm experimenting with MARP to do some markdown presentation and really like it, but recently I have been founding some problem when trying to create a presentation with some images, let me describe my file structure first:
```
marp_presentations 
├─ marp_exports
│   └─marp_example.html
├─ marp_statics
│    └─ image_example1.jpg
├─ marp_md1.md
└─ marp_md2.md
```

If I try to place a image in my .md like this:

```
![width:1000px](/marp_presentations/marp_statics/image_example1.jpg)
```

In the VSCode preview it works perfectly:
![image](https://github.com/marp-team/marp/assets/114766807/e99ee26f-b1ab-48c4-a46b-553845e7b269)

But when I export my slide deck to HTML format it doesn't even display the image.

Can someone help me to figure out this? My first guess was that it's just a path problem, but tried some things and it didn't work...

**Comment by yhatt** [comment]
`/marp_presentations/marp_statics/image_example1.jpg` is the absolute path. The resolution rule of that is different depending on the view:

- **VS Code preview**: The absolute path is resolved from the root of opening VS Code workspace.
- **Web browser (Local HTML file)**: The absolute path is resolved from the root of file system `file:///`
- **Web browser (HTTP(S) URL)**: The absolute path is resolved from the domain root of URL.
  - If hosted local web server at the workspace root, the result would be same as VS Code preview.

Due to context sensitivity, we do not recommend using the absolute path for local resources.

**Comment by igrzi** [comment, marked as answer]
Oh, I found the problem.

To display in my HTML file I needed to make the path like this:
```bash
../marp_static/image_example1.jpg
```

Instead of:

```bash
/marp_presentations/marp_statics/image_example1.jpg
```

Only downside is that now I can't see the image in my VSCode preview windows, but that's okay. Thank you so much Mr. Hattori, I'm really liking the software!
